structuralalignment/superposition/theseus.py

```diff
@@ -152,49 +105,66 @@ def _filemap(self):
 
         .. todo::
 
             Can we use a different alignment tool, preferrably in Python? E.g. biotite?
 
-        Parameters
-        ----------
+
+
+        A mode for superpositioning macromolecules with
+        identical sequences and numbers of residues,
+        for instance, multiple models in an NMR family
+        or multiple structures from different crystal
+        forms of the same protein.
+
+
+        Parameter
+        ---------
         structures : list
             list of atomium objects
+
+        **kwargs : dictionary
+            optional parameter
         """
+
         with enter_temp_directory(remove=False) as (cwd, tmpdir):
             print("DEBUG: Running in", tmpdir, " -- TODO: DELETE BEFORE MERGE --")
-            self._create_pdb(structures)
-            self._get_fasta()
-            self._concatenate_fasta()
-            self._filemap()
-            seq_alignment_output = subprocess.check_output(
-                [
-                    self.alignment_app,
-                    "-maxiters",
-                    str(self.max_iterations),
-                    "-in",
-                    self.fastafile,
-                    "-out",
-                    self.alignment_file,
-                    "-clwstrict",
-                ],
-                universal_newlines=True,
-            )
-            self._parse_alignment(seq_alignment_output)
-            print(seq_alignment_output)
-            superposition_output = self._run_theseus()
-            print(superposition_output)
-            print("-- OUTPUT WE NEED TO PARSE BEFORE DELETING TEMPFILES --")
-            print(*list(Path(tmpdir).glob("theseus_*")), sep="\n")
+
+            self.pdbs_filename = self.__create_pdb(structures)
+
+            if identical:
+                superposition_output = self._run_theseus()
+            else:
+                self._get_fasta()
+                self._concatenate_fasta()
+                self._filemap()
+                seq_alignment_output = subprocess.check_output(
+                    [
+                        self.alignment_app,
+                        "-maxiters",
+                        str(self.max_iterations),
+                        "-in",
+                        self.fastafile,
+                        "-out",
+                        self.alignment_file,
+                        "-clwstrict",
+                    ],
+                    universal_newlines=True,
+                )
+                self._parse_alignment(seq_alignment_output)
+                print(seq_alignment_output)
+                superposition_output = self._run_theseus_align()
+                print(superposition_output)
+                print("-- OUTPUT WE NEED TO PARSE BEFORE DELETING TEMPFILES --")
+                print(*list(Path(tmpdir).glob("theseus_*")), sep="\n")
         return self._parse_superposition(superposition_output)
 
     def _parse_alignment(self, output):
```

Comment:
There is more information we can parse in the output files. Let's go over the documentation and try to come up with what's interesting for us!